src/status_im/common/parallax/blacklist.cljs

```diff
@@ -0,0 +1,19 @@
+(ns status-im.common.parallax.blacklist
+  (:require
+    [native-module.core :as native-module]))
+
+(def ^:const device-id (:device-id (native-module/get-device-model-info)))
```

Comment:
Nitpick: we don't need to use `:const` metadata as much as we think we need. The only difference is that the CLJS compiler will inline `device-id` whenever it sees it. But `device-id` is currently being used only by the static value `blacklisted?`. Same for `minimum-device-code`. So it ends up being noise for no tangible value.